content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -175,6 +175,23 @@ The items listed here represent new feature flags implemented since the previous
 
 For more details on feature flags see [OpenZFS Feature Flags](https://openzfs.github.io/openzfs-docs/Basic%20Concepts/Feature%20Flags.html) and [OpenZFS zpool-feature.7](https://openzfs.github.io/openzfs-docs/man/7/zpool-features.7.html).
 
+## 23.10.1.1 Changelog
+
+**January <!--9-->, 2024**
+
+iXsystems is pleased to release TrueNAS SCALE 23.10.1.1!
+This is a small hotpatch to address issues reported by the community after the 23.10.1 release.
+
+Changes:
+
+* Fix non-physical interface (Link Aggregation, VLAN, Bridge) link address getting into database when it is updated ([NAS-125932](https://ixsystems.atlassian.net/browse/NAS-125932)).
+
+### 23.10.1.1 Ongoing Issues
+
+* TrueNAS Enterprise High Availability customers with iSCSI and ALUA enabled can experience intermittent iSCSI management issues due to operation timeout. A fix is forthcoming in the 23.10.2 release.
+
+See the **23.10.1 Ongoing Issues** list below for any additional details about issues discovered after the 23.10.1 release.
+
```

Comment:
I think we can merge the alua note with the 23.10.1 ongoing issues, remove the issue note from 23.10.1 for physical interfaces, and leave the sentence about looking at 23.10.1 ongoing issues here with the hotpatch notes. This
```suggestion
### 23.10.1.1 Ongoing Issues

* TrueNAS Enterprise High Availability customers with iSCSI and ALUA enabled can experience intermittent iSCSI management issues due to operation timeout. A fix is forthcoming in the 23.10.2 release.

See the **23.10.1 Ongoing Issues** list below for any additional details about issues discovered after the 23.10.1 release.

```